app/models/user.rb

```diff
@@ -2,8 +2,21 @@
 
 # User class
 class User < ApplicationRecord
+  has_many:jobs , dependent: :destroy
   # Include default devise modules. Others available are:
   # , :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable, :confirmable
+
+  # validation
+  VALID_PIN_REGEX = /[1-9]{1}[0-9]{5}/
+
+  validates :first_name, presence: true
```

Comment:
no need to write presence:true  for each field separately, 
validates: fields name separated by comma, presence:true

Reply:
done